src/frontend/js/widgets/Dashboard/components/DashboardItem/Enrollment/ProductCertificateFooter/index.tsx

```diff
@@ -27,33 +33,41 @@ const messages = {
   },
 };
 
+const getActiveProductOrder = (orders: OrderEnrollment[], productId: string) => {
+  const activeOrderStates = [OrderState.PENDING, OrderState.VALIDATED, OrderState.SUBMITTED];
+  const filter = (order: OrderEnrollment) =>
+    activeOrderStates.includes(order.state) && order.product === productId;
+  return orders.find(filter);
+};
```

Comment:
Could you move this function into an util module. First into the `widgets/DashboardItem` then I think it will be easier to move it elsewhere if we need it.

Reply:
I would prefere to let it here until we need it somewhere else but i moved it.